include/spatio_temporal_voxel_layer/spatio_temporal_voxel_layer.hpp

```diff
@@ -96,6 +96,8 @@ class SpatioTemporalVoxelLayer : public costmap_2d::CostmapLayer
   virtual void updateCosts(costmap_2d::Costmap2D& master_grid, int min_i, int min_j, \
                                                                int max_i, int max_j);
 
+  virtual void clearArea(int start_x, int start_y, int end_x, int end_y) override;
```

Comment:
What is this overriding?

Reply:
Are you referring at the override identifier? 
It a personal preference,  it can be removed. It tells explicitly to the compiler that is overriding a virtual function of the base class. This way it will fail in compilation is the base method doesn't have the same signature or if it doesn't exist.